mautic/mautic
# [UX] Copy code blocks on click

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

this pr introduces a new feature that allows users to easily copy the content of code blocks by simply clicking on them. 

this functionality is particularly useful when working with code snippets or examples that need to be quickly copied and pasted elsewhere

when the user clicks on the code block or the copy icon, the entire content of the block is automatically copied to the user's clipboard, ready to be pasted






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Marketplace > Open any > Click on the code for CLI then paste somewhere
